config/routes.rb

```diff
@@ -12,6 +12,7 @@
   namespace :admin do
     root to: "users#index"
     resources :users
+    resources :articles
   end
```

Comment:
They need to be under the CMS namespace, the same as the ContentPages and ContentBlocks. Otherwise these routes will show up in the public version wont they ?

I think I added this class to hide CMS routes, like the new ones to articles

  constraints CmsRouteConstraint.new do

Reply:
Well users is under admin and has been since the start.  Plan is to move content pages and blocks here also and then have a single app as this way is rather unconventional.

Reply:
They can go anywhere, but the way you've got them here does not use the CmsRouteConstraint to make sure that they will be hidden on the public site

Reply:
OK, I may need a picture drawn for me; seems like user/admin is hidden on the public site even without being in the CmsRouteConstraint.